scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -105,16 +105,14 @@ final case class UncachedFile private[scalding] (source: Either[String, URI]) {
   }
 
   private[this] def addHdfs(conf: Configuration): CachedFile = {
-    HDistributedCache.createSymlink(conf)
```

Comment:
yes we do generally try to maximize compatibility because upgrading hadoop is a painful process. When it is easy, we try to maximize compatibility.